src/actions/question.js

```diff
@@ -202,6 +202,56 @@ export function deleteQuestion(queueId, questionId) {
   }
 }
 
+/**
+ * Delete all Questions
+ */
+const deleteAllQuestionsRequest = makeActionCreator(
+  types.DELETE_ALL_QUESTIONS.REQUEST,
+  'queueId'
+)
+
+export const deleteAllQuestionsSuccess = makeActionCreator(
+  types.DELETE_ALL_QUESTIONS.SUCCESS,
+  'queueId'
+)
+
+const deleteAllQuestionsFailure = makeActionCreator(
+  types.DELETE_ALL_QUESTIONS.FAILURE,
+  'queueId'
+)
+
+export function deleteAllQuestions(queueId) {
+  return dispatch => {
+    dispatch(deleteAllQuestionsRequest(queueId))
+
+    var questionList
+    axios
+      .get(`/api/queues/${queueId}/questions`)
```

Comment:
This is going to perform poorly for queues with large numbers of remaining questions slash will run a ton of concurrent network requests. Let's implement this with a new endpoint - probably `DELETE /api/queues/###/questions`.